spec/services/broadcasts/welcome_notification/generator_spec.rb

```diff
@@ -28,11 +34,27 @@
       end.to not_change(user.notifications, :count)
     end
 
-    it "sends only 1 notification at a time" do
-      user = create(:user, :with_identity, identities: ["github"], created_at: 1.week.ago)
-      expect do
-        sidekiq_perform_enqueued_jobs { described_class.call(user.id) }
-      end.to change(user.notifications, :count).by(1)
+    it "sends only 1 notification at a time with the right notification" do # rubocop:disable RSpec/MultipleExpectations, RSpec/ExampleLength
```

Comment:
should re-enable those rubocop things at the end of the test/file i think

Reply:
It only applies to that line because I placed it at the back of a line. I've double-checked locally

> One or more cops can be disabled on a single line with an end-of-line comment.

[source](https://rubocop.readthedocs.io/en/latest/configuration/#disabling-cops-within-source-code)

Reply:
ah! woops, my mistake then :) please ignore me